rotkehlchen/constants/assets.py

```diff
@@ -146,8 +146,8 @@
 A_BSQ = Asset.initialize(
     identifier='BSQ',
     asset_type=AssetType.OTHER,
-    name="Bisq DAO Token",
-    symbol='BSQ',
+    name="BSQ",
+    symbol='Bisq DAO Token',
```

Comment:
Hello @Aragar199 ! Thank you so much for fixing the issues but there is something else. This is some mistake I made in the SQL files but is already fixed. Will tell you how to fix for your database. Copy the `rotkehlchen/data/global.db` of the repo to the `global_data` folder for rotki in your system https://rotki.readthedocs.io/en/latest/usage_guide.html#rotki-data-directory (make a backup of the old one) This db contains the fix for the BSQ asset. Then recreate this file as you did before.

You can check the diff before commiting. The only things changing should be the header of this file with the new date and at the end the A_SDL asset.

Sorry for the inconvenience

Reply:
Ah okay, no problem. It was an easy fix. Let me know if there is anything else!